frontend/src/metabase/account/settings/components/AccountHeader.styled.jsx

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import colors from "metabase/lib/colors";
+
+import {
+  breakpointMinMedium,
+  breakpointMinSmall,
+  space,
+} from "metabase/styled-components/theme";
+import UserAvatar from "metabase/components/UserAvatar";
+
+export const AccountHeaderRoot = styled.div`
```

Comment:
We seem to be using "Root" and "Container", even, in the past, "Wrapper" for the same objective.

My personal preference is "Container" — certainly just because I'm used to it.

Wdyt of us all settling on one? Any preference?

Reply:
I would go with `Root` since it matches React terminology (root node) and it's shorter. But I'm happy to accept any convention that the team would be happy with. Might worth opening an RFC for this.